test/controllers/EditSetRecordControllerSpec.scala

```diff
@@ -2601,6 +2624,7 @@ class EditSetRecordControllerSpec extends BaseSpec {
     ExpectedEditRecordPage(
       title = "Edit record",
       heading = s"TNA reference: ${editSetRecord.ccr}",
+      headingPacId = s"PAC-ID: ${editSetRecord.oci} Physical Record",
```

Comment:
This wouldn't work for a record without a type (e.g. `COAL.2022.V2RJW`).

Reply:
For `COAL.2022.V2RJW` it works by checking for this value `subHeading = s"PAC-ID: $oci"`instead as seen in test _"all data is valid with no record type suffix"_

Reply:
Yes, I see that it works. But by coincidence.

However, if you used that same record and asserting something different, it would fail.

If you wanted to fix it, you could dynamically determine the correct type.

Reply:
Done